docs/index.rst

```diff
@@ -113,6 +113,11 @@ To enable this hook, add it as the ``tm.manager_hook`` setting in your app.
        config.include('pyramid_tm')
        # ...
 
+This specific example, using an explicit mode non-threadlocal manager, is
+highly recommended and is shipped as :func:`pyramid_tm.explicit_manager`.
+Simply set ``tm.manager_hook = pyramid_tm.explicit_manager`` in your settings
```

Comment:
I mean... sure. In general it's nicer to people if we put the functionality in early under a flag and warn about the default switching and then the major version bump is for actually changing the default. It has a much higher chance of not breaking the world to do things this way when possible but obviously much more work to develop. I'd definitely want to take this approach to switching the manager.

The issue with switching the manager is that if people are using the threadlocal one there will be permanent implicit transactions/sessions that are never closed / committed / aborted. They would just turn into silent memory leaks with no clear reason.

The tween changes we made are different in that they only affect exception situations... there's no "major" change in there for requests that don't raise.